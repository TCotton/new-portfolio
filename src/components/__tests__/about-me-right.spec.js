import React from 'react';
import renderer from 'react-test-renderer';

import AboutMeRight from '../about-me-right';

describe('AboutMeRight', () => {

	let component, instance;

	beforeEach( () => {
		component = renderer.create(
			<AboutMeRight />
		);
		instance = component.getInstance();
	});

	it('renders correctly', () => {
		const tree = renderer
			.create(<AboutMeRight />)
			.toJSON()
		expect(tree).toMatchSnapshot();
	})

	it('should call shouldComponentUpdate', () => {
		const shouldUpdate = instance.shouldComponentUpdate();
		expect(shouldUpdate).toBe(true);
	});
})
