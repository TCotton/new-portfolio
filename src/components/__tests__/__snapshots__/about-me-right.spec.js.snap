// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMeRight renders correctly 1`] = `
<div
  className="right-small"
>
  <h5
    className="content-title"
  >
    My CV and online profiles
  </h5>
  <p>
    Please read my Stack Overflow Careers page:
    <br />
    <a
      className="underline"
      href="https://stackoverflow.com/story/andyw"
      rel="external"
    >
      https://stackoverflow.com/story/andyw
    </a>
  </p>
  <a
    href="https://twitter.com/andywalpole"
  >
    <p>
      <img
        alt="Twitter account of Andy Walpole"
        src="/images/svg/twitter.svg"
      />
    </p>
  </a>
  <p>
    <img
      alt="yours truly, the author of this portfolio and blog"
      src="/images/andyw.jpg"
    />
  </p>
</div>
`;
